src/main/scala/org/broadinstitute/dsde/agora/server/webservice/ServiceHandlerProps.scala

```diff
@@ -0,0 +1,10 @@
+
+package org.broadinstitute.dsde.agora.server.webservice
+
+import akka.actor.Props
+
+trait ServiceHandlerProps {
```

Comment:
Good question. It was separate before so I'm not sure what the initial reason was. (I think it was a pattern we had used before in Apollo)